scripts.go

```diff
@@ -116,7 +116,8 @@ trap 'rm -f /tmp/.git_ssh.$$' 0
 
 if [ "$1" = "-i" ]; then
     SSH_KEY=$2; shift; shift
-    echo "ssh -i $SSH_KEY \$@" > /tmp/.git_ssh.$$
+    echo "#!/bin/%v \n \
```

Comment:
With `%v`, another parameter is introduced so you need to modify line 130/131 accordingly.

You'll also need to ensure the test passes.